protos/feast/core/CoreService.proto

```diff
@@ -274,3 +281,42 @@ message StopIngestionJobRequest {
 
 // Request from stopping an ingestion job
 message StopIngestionJobResponse {}
+
+message GetFeatureStatisticsRequest {
+    // Feature set to retrieve the statistics for. A fully qualified feature set
+    // id in the format of project/feature_set:version must be provided.
+    string feature_set_id = 1;
+
+    // Optional filter which filters returned statistics by selected features. These
+    // features must be present in the data that is being processed.
+    repeated string feature_ids = 2;
```

Comment:
Feature references?

Reply:
feature names actually

Reply:
Alright, so we change it to feature_names then?

Reply:
I guess it doesn't matter much since we can always change a field name

Reply:
I think since we're providing `feature_set_id` `features` should logically refer to feature names within that feature set.